pyanaconda/modules/storage/platform.py

```diff
@@ -142,14 +145,24 @@ def stage1_descriptions(self):
             "mdarray": _(RAID_DESCRIPTION)
         }
 
+    @property
+    def stage1_constraints(self):
+        """The platform-specific constraints for the stage1 device."""
+        constraints = {
+            PLATFORM_DEVICE_TYPES: ["disk"]
+        }
+        return dict(super().stage1_constraints, **constraints)
+
+    @property
+    def _bootloader_partition(self):
+        """The default bootloader partition for this platform."""
+        return PartSpec(
+            fstype="biosboot",
+            size=Size("1MiB")
+        )
```

Comment:
I'm not quite sure why this private method is part of this commit. Seems to me it's not used anywhere in this commit.

Reply:
Fixed.